pkg/render/intrusion_detection.go

```diff
@@ -1440,6 +1441,19 @@ func (c *intrusionDetectionComponent) adDetectorAccessRole() *rbacv1.Role {
 					"update",
 				},
 			},
+			{
+				APIGroups: []string{
+					ADResourceGroup,
+				},
+				Resources: []string{
+					ADLogTypeMetaDataResourceName,
```

Comment:
I think you could simply add "ADLogTypeMetaDataResourceName" to line 1435 and omit this block.